view/src/components/account.js

```diff
@@ -0,0 +1,347 @@
+  },
+  locationText: {
+    paddingLeft: "16px",
+  },
+  buttonProperty: {
+    position: "absolute",
+    top: "50%",
+  },
+  uiProgess: {
+    position: "fixed",
+    zIndex: "1000",
+    height: "32px",
+    width: "32px",
+    left: "50%",
+    top: "35%",
+  },
+  progess: {
+    position: "absolute",
+  },
+  uploadButton: {
+    marginLeft: "12px",
+    margin: theme.spacing(1),
+  },
+  customError: {
+    color: "red",
+    fontSize: "0.8rem",
+    marginTop: 112,
+  },
+  submitButton: {
+    marginTop: "12px",
+  },
+});
+
+class Account extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      firstName: "",
+      lastName: "",
+      email: "",
+      phoneNumber: "",
+      username: "",
+      country: "",
+      profilePicture: "",
+      uiLoading: true,
+      buttonLoading: false,
+      imageError: "",
+    };
+  }
+
+  componentDidMount = () => {
+    authMiddleWare(this.props.history);
+    const authToken = localStorage.getItem("AuthToken");
+    axios.defaults.headers.common = { Authorization: `${authToken}` };
+    axios
+      .get("/user")
+      .then((response) => {
+        this.setState({
+          firstName: response.data.userCredentials.firstName,
+          lastName: response.data.userCredentials.lastName,
+          email: response.data.userCredentials.email,
+          phoneNumber: response.data.userCredentials.phoneNumber,
+          country: response.data.userCredentials.country,
+          username: response.data.userCredentials.username,
+          uiLoading: false,
+        });
+      })
+      .catch((error) => {
+        if (error.response.status === 403) {
+          this.props.history.push("/login");
+        }
+        console.error(error);
+        this.setState({ errorMsg: "Error in retrieving the data" });
+      });
+  };
+
+  handleChange = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value,
+    });
+  };
+
+  handleImageChange = (event) => {
+    this.setState({
+      image: event.target.files[0],
+    });
+  };
+
+  profilePictureHandler = (event) => {
+    event.preventDefault();
+    this.setState({
+      uiLoading: true,
+    });
+    authMiddleWare(this.props.history);
+    const authToken = localStorage.getItem("AuthToken");
+    let form_data = new FormData();
+    form_data.append("image", this.state.image);
+    form_data.append("content", this.state.content);
+    axios.defaults.headers.common = { Authorization: `${authToken}` };
+    axios
+      .post("/user/image", form_data, {
+        headers: {
+          "content-type": "multipart/form-data",
+        },
+      })
+      .then(() => {
+        window.location.reload();
+      })
+      .catch((error) => {
+        if (error.response.status === 403) {
+          this.props.history.push("/login");
+        }
+        console.error(error);
+        this.setState({
+          uiLoading: false,
+          imageError: "Error in posting the data",
```

Comment:
No need to change this now, but in case you're curious: usually error messages like this are not passed as human-readable text, but as an identifier of some kind.  This is especially important if you need to localize your app to work in multiple languages, since otherwise you'd have to list all of the translations for every error message whenever you generate an error.